src/Symfony/Component/Finder/Finder.php

```diff
@@ -136,13 +136,89 @@ public function depth($level)
      *
      * @return $this
      *
+     * @deprecated since 4.1, to be removed in 4.2
      * @see strtotime
      * @see DateRangeFilterIterator
      * @see DateComparator
      */
     public function date($date)
     {
-        $this->dates[] = new Comparator\DateComparator($date);
+        $this->dateModified($date);
+
+        return $this;
+    }
+
+    /**
+     * Adds tests for file dates (last accessed).
+     *
+     * The date must be something that strtotime() is able to parse:
+     *
+     *   $finder->date('since yesterday');
+     *   $finder->date('until 2 days ago');
+     *   $finder->date('> now - 2 hours');
+     *   $finder->date('>= 2005-10-15');
+     *
+     * @param string $date A date range string
+     *
+     * @return $this
+     *
+     * @see strtotime
+     * @see DateRangeFilterIterator
+     * @see DateComparator
+     */
+    public function dateAccessed($date)
+    {
+        $this->dates[] = new Comparator\DateComparator($date, Comparator\DateComparator::TIME_TYPE_ACCESSED);
+
+        return $this;
+    }
+
+    /**
+     * Adds tests for file dates (last changed).
```

Comment:
We should add a super brief explanation of the difference between "last changed" and "last modified". Thanks!

Reply:
Added sort description:

```
 +     * Changed date is the last time the file was:
 +     *
 +     *   written
 +     *   permissions changed
 +     *   moved
 +     *   renamed
```